core/src/main/java/org/kabeja/entities/Text.java

```diff
@@ -473,8 +474,17 @@ public void transform(TransformContext context) {
             }
       
     }
-    
-    
+
+    /**
+     * @ToDo: implement this method
+     *
+     * This is a planar entity, therefore this method should be implemented,
+     * otherwise OCS-coordinates will be used regardless of calling this method.
+     */
+    @Override
+    public void toWcs(){
+        throw new NotImplementedException();
```

Comment:
@JohannesMessner I love this not implemented exception! We should just throw this everywhere:

User:          `start crowd:it`
crowd:it:    `NotImplementedException`